src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -148,26 +160,27 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
         className={noTranslate ? 'no-translate' : undefined}
         dir={dir}
         endAdornment={(
-          <InputAdornment position="end">
-            <Tooltip title={showPassword ? getTranslation(translations, 'hide') : getTranslation(translations, 'show')}>
-              <IconButton
-                aria-label={getTranslation(translations, 'visibilityToggleLabel')}
-                aria-pressed={showPassword}
-                aria-controls={name}
-                onClick={handleClickShowPassword}
-                onMouseDown={handleMouseDownPassword}
```

Comment:
@shawyu-okta  @glenfannin-okta This `mouseDown` handler was the culprit for the cursor jumping to the front of the input. I spoke to @leemchale-okta because he originally added this handler since it used to be present on the Odyssey password component, but Lee was able to verify that the new Odyssey component doesn't have this handler anymore